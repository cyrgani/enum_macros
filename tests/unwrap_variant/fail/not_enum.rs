use enum_macros::UnwrapVariant;

#[derive(UnwrapVariant)]
struct S {
    #[unwrap(ref)]
    a: u32,
    b: u8,
}

#[derive(UnwrapVariant)]
union U {
    #[unwrap(ref, mut)]
    a: u32,
    b: f32,
}

fn main() {}