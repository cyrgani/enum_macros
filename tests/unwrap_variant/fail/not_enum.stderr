error: expected `enum`
 --> tests/unwrap_variant/fail/not_enum.rs:4:1
  |
4 | struct S {
  | ^^^^^^

error: expected `enum`
  --> tests/unwrap_variant/fail/not_enum.rs:11:1
   |
11 | union U {
   | ^^^^^
